servers/services-bench/README.md

```diff
@@ -0,0 +1,28 @@
+# Nessie VersionStore micro benchmarks
+
+Building:
+```bash
+./gradlew :nessie-services-bench:jmhJar
+```
+
+Running:
+```bash
+java -jar servers/services-bench/build/libs/nessie-services-bench-*-jmh.jar
+```
+
+## Async-profiler
+
+See the [Async Profiler repo](https://github.com/async-profiler/async-profiler) for a pre-built library or how to build
+it from source.
+
+Running (Linux):
+```bash
+LD_LIBRARY_PATH=(PATH-TO-LIBRARY)/ java \
+  -jar servers/services-bench/build/libs/nessie-services-bench-*-jmh.jar \
+  -prof async
+```
+
+## Linux Perf tools
+
+Install the appropriate `linux-tools` package for your distribution, or `make` it from the Linux sources in `tools/perf`
```

Comment:
From what I could find, the `linux-tools` package is a Ubuntu thing. Maybe we could rephrase this paragraph and focus more on which commands need to be available, rather than which packages contain those commands.

Reply:
Rephrased